File: i-ascii-unicode/python/character.py

def main():

    sym = 'g'
    print(sym)

    # character_value to ascii_value
    ch = 'g'
    print(f"The ascii value of {ch} is : {ord(ch)}")

    # ascii_value to character_value
    num = 103
    print(f"The character value of {num} is : {chr(num)}")

    # escape sequence character
    a = 'P'
    b = '\t'
    c = 'R'
    d = '\t'
    e = 'A'
    user_name = a + b + c + d + e + '\n' + f"my name start with (pra + kash)"
    print(user_name)

    return

main()

'''
    output
    -------
    g
    The ascii value of g is : 103
    The character value of 103 is : g
    P       R       A
    my name start with (pra + kash)
    
'''

File: i-ascii-unicode/javascript/character.js

function main(){

    let sym = 'g'; 
    console.log(sym);

    // character_value to ascii_value
    let ch = 'g';
    console.log(`The ascii value of ${ch} is : ${ch.codePointAt()}`);

    // ascii_value to character_value
    let num = 103;
    console.log(`The character value of ${num} is : ${String.fromCodePoint(num)}`);

    // escape sequence character
    let a = 'P';
    let b = '\t';
    let c = '\R';
    let d = '\t';
    let e = 'A';
    let user_name = a + b + c + d + e + '\n' + "my name start with (pra + kash)";
    console.log(user_name);

    return;
}
main();

/*
    output
    ---------
    g
    The ascii value of g is : 103
    The character value of 103 is : g
    P       R       A
    my name start with (pra + kash)

*/